crates/cairo-lint-core/tests/test_files/ifs/ifs_same_cond

```diff
@@ -0,0 +1,167 @@
+//! > Same condition with boolean
```

Comment:
add a tests with:
- more than one else if
- one where 2 conditions are similar but not the last one
and try to think about more edge cases to avoid bugs